app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,12 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".CreateSighting"></activity>
+        <activity android:name=".CreateSighting"
+            android:parentActivityName=".MainActivity"/>
+
+        <activity android:name=".ViewSighting"
+            android:parentActivityName=".MainActivity"
+            android:launchMode="singleTop"></activity>
     </application>
 
 </manifest>
```

Comment:
Please add an extra empty line at the end

Reply:
I fixed it